src/Foundatio/Caching/HybridCacheClient.cs

```diff
@@ -88,19 +88,17 @@ public class HybridCacheClient : ICacheClient {
             CacheValue<T> cacheValue;
             bool requiresSerialization = TypeRequiresSerialization(typeof(T));
```

Comment:
I would agree.  It's currently only used for the logging.  If that isn't needed, we can remove it.  I'll update the PR.